src/scripts/manage_artifacts/artifacts.py

```diff
@@ -0,0 +1,67 @@
+import os
+import argparse
+
+cfn_bucket = 'sceptre-cloudformation-bucket-bucket-65ci2qog5w6l'
+repo_name = 'BridgeDownstream'
+
+
+def read_args():
+  descriptions = '''
+  Uploading to S3 and deletion from S3 of scripts and templates.
+  '''
+  parser = argparse.ArgumentParser(
+    description='')
+  parser.add_argument(
+    '--namespace',
+    default='bridge-downstream-dev')
+  group = parser.add_mutually_exclusive_group(required=True)
+  group.add_argument('--upload', action='store_true')
+  group.add_argument('--remove', action='store_true')
+  group.add_argument('--list', action='store_true')
+  args = parser.parse_args()
+  return args
+
+
+def upload(namespace):
+  '''Copies Glue scripts and CFN templates to the artifacts bucket'''
+  scripts_local_path = 'src/glue/'
+  scripts_s3_path = f's3://{cfn_bucket}/{repo_name}/{namespace}/glue/'
+  cmd = f'aws s3 sync {scripts_local_path} {scripts_s3_path}'
+  print(f'Invoking command: {cmd}')
+  os.system(cmd)
+
+  templates_local_path = 'templates/'
+  templates_s3_path = f's3://{cfn_bucket}/{repo_name}/{namespace}/templates/'
+  cmd = f'aws s3 sync {templates_local_path} {templates_s3_path}'
+  print(f'Invoking command: {cmd}')
+  os.system(cmd)
+
+
+def delete(namespace):
+  '''Removes all files recursively for namespace'''
+  s3_path = f's3://{cfn_bucket}/{repo_name}/{namespace}/'
+  cmd = f'aws s3 rm --recursive {s3_path}'
+  print(f'Invoking command: {cmd}')
+  os.system(cmd)
+
+
+def list_namespaces():
+  '''List all namespaces'''
+  s3_path = f's3://{cfn_bucket}/{repo_name}/'
+  cmd = f'aws s3 ls {s3_path}'
+  print(f'Invoking command: {cmd}')
+  os.system(cmd)
+
+
+def main(namespace):
+  namespace = args.namespace
```

Comment:
Where is `args` coming from? Should the function parameter be `args`?

Reply:
that was a woops -- thanks for pointing it out